src/design-tokens/ThemeProvider.tsx

```diff
@@ -3,6 +3,8 @@ import { ThemeProvider as MuiThemeProvider } from '@material-ui/core/styles';
 import { ThemeProvider as EmotionThemeProvider } from 'emotion-theming';
 import i18next from 'i18next';
 
+import loadDayJSLocale from '../App/load-dayjs-locale';
```

Comment:
maybe now it would be better if we move `load-dayjs-locale `file to design-tokens 😉